### PR TITLE
Connections to Points

### PR DESCRIPTION
**Pull request by RyanDavies19**
I have changed all references to connections to points in the code per @mattEhall's instructions. Going forward everything should change to Points, connections is the old word. Everything should still work with input files that use "Connection/Con/C" but internally everything is points now. If I missed anything please add in changes.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -20,7 +20,7 @@ list(APPEND EXTRA_INCLUDES "${PROJECT_SOURCE_DIR}/Source")
 set(BUILD_DOCS OFF CACHE BOOL
     "Build the documentation for users and developers")
 set(BUILD_BENCHMARKS OFF CACHE BOOL
-    "Build the benchmarks. Makes use of Google's Benchmark library and requires an internet connection to download.")
+    "Build the benchmarks. Makes use of Google's Benchmark library and requires an internet point to download.")
```
Comment (sanguinariojoe) [inline CMakeLists.txt:23]: revert this please

File: extern/Catch2/LICENSE.txt [hunk 2]
```diff
@@ -19,5 +19,5 @@ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 FITNESS FOR A PARTICULAR PURPOSE, TITLE AND NON-INFRINGEMENT. IN NO EVENT
 SHALL THE COPYRIGHT HOLDERS OR ANYONE DISTRIBUTING THE SOFTWARE BE LIABLE
 FOR ANY DAMAGES OR OTHER LIABILITY, WHETHER IN CONTRACT, TORT OR OTHERWISE,
-ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ARISING FROM, OUT OF OR IN POINT WITH THE SOFTWARE OR THE USE OR OTHER
```
Comment (sanguinariojoe) [inline extern/Catch2/LICENSE.txt:22]: revert this please

File: extern/Catch2/docs/test-fixtures.md [hunk 3]
```diff
@@ -10,9 +10,9 @@ class UniqueTestsFixture {
   private:
    static int uniqueID;
   protected:
-   DBConnection conn;
+   DBPoint point;
```
Comment (sanguinariojoe) [inline extern/Catch2/docs/test-fixtures.md:13]: Revert all the changes in this file.

**Review by sanguinariojoe — COMMENTED** [review]
Low hanging fruit

File: extern/Catch2/docs/test-fixtures.md (hunk 3 above)
Comment (RyanDavies19) [reply extern/Catch2/docs/test-fixtures.md]: Reverted all three of these. Must have forgotten to ignore them when I was searching.

**Comment by mattEhall** [comment]
Thanks for catching those @sanguinariojoe :)

**Comment by sanguinariojoe** [comment]
Rebased and merged!

@RyanDavies19 , your v2 branch is not diverging from the upstream. I suggest you to reset it and then rebase it:

```
git reset --hard 4561d264217ba725c150f34bb4babbaa42b313db
git fetch upstream
git merge upstream/v2
git push -f origin v2
```

Note: I usually prefer to work on different branches that will be discarded. That way you only need to sync the v2 branch when you want to start working and move to a different branch to implement things:

```
git checkout v2
git fetch upstream
git merge upstream/v2
git push origin v2
git checkout -b new_branch_to_work
```

**Pull request merged**